src/libraries/System.Text.Json/gen/TypeGenerationSpec.cs

```diff
@@ -221,6 +229,11 @@ private bool FastPathIsSupported()
         {
             if (ClassType == ClassType.Object)
             {
+                if (ExtensionDataPropertyTypeSpec != null)
```

Comment:
Why can't the FastPath support this?

Reply:
Extension data properties (dictionaries) can have `typeof(object)` values which is a polymorphic scenario currently not supported in fast-path. Also, the ext. properties can be `JsonObject` - I spent some time prototyping support for this but the PR started getting too complex and large.

We always had `JsonExtensionData` as "not supported in fast path for 6.0" - https://github.com/dotnet/runtime/issues/51945#issuecomment-827795787. Would have be nice to get it in now, but I don't think it's worth the extra complexity at this stage.

Reply:
Do you think it makes sense to log this as an issue, explaining the limitations and concerns? At least then it will be written down somewhere.

Reply:
Yup I opened https://github.com/dotnet/runtime/issues/58943.